manifests/config.pp

```diff
@@ -1,7 +1,7 @@
 #
 define syslog_ng::config (
-  $content = '',
```

Comment:
I did a quick search in our data and I didn't find any resource with empty `content`.
But agreed : it doesn't make sense to have an empty one.